### ISSUE TITLE
GetFocussedSession crash

### ISSUE THREAD

**Issue by VolcanicArts** [DubyaDude/WindowsMediaController#12]
When using Chrome and YouTube (presumed to be similar on other browsers) when the video changes `GetFocussedSession()` is wrong as Chrome removes itself as a media source when unloading the page and re-adds itself as a media source when loading which results in an exception.

```
System.Collections.Generic.KeyNotFoundException: The given key 'chrome.exe' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at WindowsMediaController.MediaManager.GetFocusedSession(GlobalSystemMediaTransportControlsSessionManager sender)
```

I'd probably expect that if no focussed sessions were found, I.E, there are none present or the one trying to be found is null, it tries to default to another session or just returns a null controller.

**Comment by DubyaDude** [comment]
Hi, can you please provide a sample of what it prints before it crashes using the CMD sample? I'm unable to reproduce it on my end.

Me going through a few videos in chrome on Youtube:
![image](https://user-images.githubusercontent.com/13679909/208253365-db11b09e-27a8-489a-9acc-699b1979cd21.png)

**Comment by VolcanicArts** [comment]
I can't recreate it with the sample CMD because the sample CMD doesn't use `GetFocussedSession()`

**Comment by DubyaDude** [comment]
When are you executing GetFocusedSession? Sample uses the `OnFocusedSessionChanged` event

**Comment by VolcanicArts** [comment]
I'm using it outside of the events so that I can control the current session

**Comment by DubyaDude** [comment]
Mhm, was adding the check. Was just curious as to how you got it.
Will be fixed in the next release.

**Comment by VolcanicArts** [comment]
https://github.com/VolcanicArts/VRCOSC/blob/dev/VRCOSC.Game/Modules/Modules/Media/MediaModule.cs#L54

Here's one example. That code isn't called from any of the events of the WMC library

**Comment by DubyaDude** [comment]
Yup, to correct my wording, was curious how you got the exception*
What you stated earlier about grabbing it from non-event cleared that up for me.

**Comment by DubyaDude** [comment]
Fixed in 2.4.0 release
